plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/Messages.properties

```diff
@@ -127,6 +127,7 @@ Warnings.RFLint.ParserName=Robot Framework Lint
 Warnings.LinuxKernelOutput.ParserName=Linux Kernel Output Parser
 Warnings.WarningsPlugin.ParserName=Warnings Plugin Native Format
 Warnings.ProtoLint.ParserName=ProtoLint
+Warnings.Trivy.ParserName=aquasec trivy vulnerability scanner
```

Comment:
Labels should be capitalized.

I still think that the text in Jenkins side panel would look better if you use *Trivy* rather than *Aquasec Trivy Vulnerability Scanner*.